modules/components/pages/processors/aws_bedrock_chat.adoc

```diff
@@ -0,0 +1,236 @@
+= aws_bedrock_chat
+:type: processor
+:status: experimental
```

Comment:
aren't we classifying this as enterprise?

Reply:
Yes, has been added to the table and the enterprise admonition appears on page.